pkg/metadatastorage/attestationcollection/parserstorer.go

```diff
@@ -58,12 +58,21 @@ func (parsedCollection ParsedCollection) Store(ctx context.Context, tx *ent.Tx,
 	}
 
 	for _, a := range parsedCollection.Attestations {
-		if err := tx.Attestation.Create().
+		newAttestation, err := tx.Attestation.Create().
 			SetAttestationCollectionID(collection.ID).
 			SetType(a.Type).
-			Exec(ctx); err != nil {
+			Save(ctx)
+		if err != nil {
 			return err
 		}
+
+		// we parse if a parser is available. otherwise, we ignore it if no parser is available.
```

Comment:
We should probably do the same here: https://github.com/in-toto/archivista/blob/2eab1165895f7871a6ab706354fd62a13cfe88a7/pkg/metadatastorage/sqlstore/store.go#L108.

Reply:
I think that's a good idea. Are you comfortable doing that as a subsequent patch?